bin/compile

```diff
@@ -16,7 +16,7 @@ for BUILDPACK in $(cat $1/.buildpacks); do
   fi
 
   if [ "$url" != "" ]; then
-    echo "=====> Downloading Buildpack: $url"
+    echo "=====> Downloading Buildpack: $url version: ${branch:-master}"
```

Comment:
When the branch isn't specified, the repo's default branch is used, which might not necessarily be `master` (particularly with the recent shift towards `main` as the default branch). As such, I think it might be easier to just display the original full URL here rather than having to recombine the URL parts.

```suggestion
    echo "=====> Downloading Buildpack: ${BUILDPACK}"
```

Reply:
Sounds perfect for me!
Thanks